environment.yml

```diff
@@ -35,3 +35,5 @@ dependencies:
       - python-dotenv==0.15.0
       - hydra-core==1.0.5
       - natsort==7.1.0
+      - pyrr==0.10.3
+      - pyrsistent==0.16.0
```

Comment:
these are the moab imports right?

Reply:
Yes, Moab imports.  Will remove them from here. It is mostly for python simulators that facilitate matrix  and quaternion computations for robotics purposes. Such simulators will seldom have need of a ddm, we just testing it here.